chronicle/ContentView.swift

```diff
@@ -8,14 +8,123 @@
+                                Task {
+                                    await enrollmentViewModel.enroll()
+                                }
+                            }, label: {
+                                Text("Enroll Device")
+                                    .foregroundColor(.white)
+                                    .padding(10)
+                            })
+                            .background(Color(red: 109/255, green: 73/255, blue: 254/255, opacity: 1.0))
+                            .cornerRadius(8)
+                            .disabled(enrollmentViewModel.enrolling == true)
+                        }
+                    }
+                    if enrollmentViewModel.showEnrollmentError {
+                        Text("Failed to enroll device. Please double check that the Organization ID, Study ID and the Participant ID are correct. If the problem persists, please contact your study administrator.")
+                            .padding(.top)
+                    }
+                    Spacer()
+                }
+                .padding(.horizontal)
+                .textFieldStyle(.roundedBorder)
+            }
+        }
+        else {
+            ScrollView {
+                VStack(alignment: .leading) {
+                    Text("Your device has been enrolled! Thank you for participating in our study!\n\nYou have successfully installed Chronicle and may now close the application. Chronicle will continue to function as a background process on your device while the application is closed. To stop sending information to the researchers - for instance, at the end of the study - simply uninstall this app. If you have questions or concerns, please contact a member of your research team.")
+                    // later, add "Done" button which navigates to home page
+                    Spacer()
+                }.padding()
+            }
+        }
+    }
+}
+
+struct RadioButtonGroup: View {
+    @Binding var hasOrgId: Bool
+    
+    let filled = "circle.inset.filled"
+    let empty = "circle"
+    
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        LazyVGrid(columns: [GridItem(.fixed(30)), GridItem()]) {
+            LazyVGrid(columns: [GridItem()], alignment: .leading) {
+                Button(action: {
+                    hasOrgId = true
+                }, label: {
+                    Image(systemName: hasOrgId ? filled : empty)
+                })
+                Spacer()
+                Button(action: {
+                    hasOrgId = false
+                }, label: {
+                    Image(systemName: !hasOrgId ? filled : empty)
+                })
+            }
+            LazyVGrid(columns: [GridItem()], alignment: .leading) {
+                Text("Yes")
+                Spacer()
+                Text("No")
+            }
+        }
+        .foregroundColor(Color(red: 109/255, green: 73/255, blue: 254/255, opacity: 1.0))
```

Comment:
Consider defining colors in "Assets"
<img width="230" alt="Screen Shot 2021-08-13 at 11 58 40 AM" src="https://user-images.githubusercontent.com/15723547/129406374-512dcbf0-4d86-49b9-9aed-3957edb0ad95.png">

You can customize each color for both Dark and light appearance by editing modifying the RGB values. This would make it easier to reuse colors in other views
For example, if you have a color named "Primary", you can reference it in the UI as Color("Red"). Even better, you can extend the Color struct and define the custom color values. 

```
// Colors.swift

import SwiftUI
extension Color {
    static var primaryColor: Color {
        Color("Primary")
    }
    // add other colors here
}

````

The in the UI, you could simply do 
```
Text("sample text)
 .foregroundColor(Color(.primary))
```
(No need to type `Color.primary`)

Reply:
this is awesome! thanks so much for teaching me.